test_app/CMakeLists.txt

```diff
@@ -13,12 +13,13 @@ if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_GREATER_EQUAL "4.4")
 endif()
 if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_LESS "4.4")
     set(EXCLUDE_COMPONENTS "es8311")
+    set(EXCLUDE_COMPONENTS "es7210")
```

Comment:
are are resetting the EXCLUDE_COMPONENTS variable here.

Could you add it to the same line as es8311?

Reply:
Done. Sorry for the mistake.